canonical-websites/www.ubuntu.com
# Update terms-and-policies with partner portal

## Done

- Added the partner portal terms to the overview page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Compare to the [copy doc](https://docs.google.com/document/d/1p3ZWroZ3nkw8XJJdPvPxSlWGYcLZLT8UbUVuppBcyHk/edit#)


## Issue / Card

Fixes #4843

## Screenshots

![image](https://user-images.githubusercontent.com/441217/54481213-b2ac3500-4829-11e9-9898-36c722126eed.png)
